src/common/file_system/local_file_system.cpp

```diff
@@ -220,10 +220,10 @@ void LocalFileSystem::createDir(const std::string& dir) const {
             // LCOV_EXCL_STOP
         }
         auto directoryToCreate = dir;
-        if (directoryToCreate.ends_with('/')) {
+        if (directoryToCreate.ends_with('/') || directoryToCreate.ends_with('\\')) {
```

Comment:
If the `\\` is only used for windows, i recommend put the prediction inside the windows macro `#ifdef _WIN32`

Reply:
wrapped this with macro